packages/dataviews/src/components/dataviews-filters/filter-summary.tsx

```diff
@@ -235,12 +238,25 @@ export default function FilterSummary( {
 	const filterInView = view.filters?.find(
 		( f ) => f.field === filter.field
 	);
+
 	const activeElements = filter.elements.filter( ( element ) => {
 		if ( filter.singleSelection ) {
 			return element.value === filterInView?.value;
 		}
 		return filterInView?.value?.includes( element.value );
 	} );
+
+	if (
+		activeElements.length === 0 &&
+		filter.type === 'integer' &&
```

Comment:
Why are we hard-coding the type here?

Reply:
See below. https://github.com/Automattic/wp-calypso/pull/103276#discussion_r2090350760